components/badges/badges.css

```diff
@@ -0,0 +1,49 @@
+@import url("../avatar/avatar.css");
+
+.badge-container {
+	display: flex;
+	align-items: baseline;
+}
+
+.badge {
+	position: relative;
+	margin-right: 1.5rem;
+}
+
+.status-badge {
+	width: 18.5%;
+	height: 19.5%;
+	position: absolute;
+	border-radius: 50%;
+	bottom: 8%;
+	right: 16%;
+	border: 1.3px solid #fff;
+}
+
+.online {
+	background-color: #4caf50;
+}
+
+.offline {
+	background-color: grey;
+}
+
+.dnd {
+	background-color: #f44336;
+}
+
+.fas {
+	font-size: 4rem;
+}
+
+.badge-icon {
+	color: #fff;
```

Comment:
You can use the CSS Sorter extension on VS code by Zascal to sort your CSS properties by alphabetical order so it helps in easier review and debugging

Reply:
I have installed and sorted my CSS propeties. Thank you Raksha for this extension.